### PR TITLE
Removed repeated dependency in reqirements.txt.

### PR DESCRIPTION
**Pull request by vignesh-cloud-prog**
Signed-off-by: vignesh-cloud-prog <vigneshun80@gmail.com>

### CODE REVIEW

**Comment by sbs2001** [comment]
What is repeated ? Could you elaborate ? Your commit message is also misleading, you are adding deps not removing them.

File: requirements.txt [hunk 1]
```diff
@@ -37,7 +37,6 @@ pygit2==1.5.0
 Pygments==2.6.1
 pyparsing==2.4.5
 pytest==5.3.2
-django-widget-tweaks==1.4.8
```
Comment (sbs2001) [inline requirements.txt:40]: This is required for the UI

**Review by sbs2001 — CHANGES_REQUESTED** [review]
This is required for the UI

File: requirements.txt (hunk 1 above)
Comment (vignesh-cloud-prog) [reply requirements.txt:40]: I am sorry about that

**Pull request closed without merging**